.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
 matrix:
   fast_finish: true
   allow_failures:
+    # For now it doesn't work in Glimmer 2. We'll try to fix before the release.
+    - env: EMBER_TRY_SCENARIO=ember-beta
```

Comment:
@MiguelMadero should we go ahead and remove this now that v0.1.3 is out? I would assume so -that way we could find out it breaks/ or works when ember v2.11.0-beta starts shipping soon :)